ResearchKit/Common/ORKScaleSlider.m

```diff
@@ -96,10 +164,9 @@ - (void)updateValueForTouchAtPoint:(CGPoint)touchPoint {
 }
 
 static CGFloat kLineWidth = 1.0;
-- (void)drawRect:(CGRect)rect
-{
-    CGRect trackRect = [self trackRectForBounds:[self bounds]];
-    CGFloat centerY = [self bounds].size.height / 2.0;
+- (void)drawRect:(CGRect)rect {
+    CGRect trackRect = [self trackRectForBounds:self.bounds];
+    CGFloat centerY = self.bounds.size.height / 2.0;
```

Comment:
Ah, I see. I replaced it to a *property getter call* because of `UIView` public documentation defining it as a *property*. I'll change it back.